aiohttp/protocol.py

```diff
@@ -173,7 +175,7 @@ def __call__(self, out, buf):
             raw_data = yield from buf.readuntil(
                 b'\r\n\r\n', self.max_headers)
         except errors.LineLimitExceededParserError as exc:
-            raise errors.LineTooLong(exc.limit) from None
+            raise errors.LineTooLong('request header', exc.limit) from None
```

Comment:
It's not, and the call here is wrong. Check out the constructor of `LineTooLong` (didn't change that one), it expects a "reason" and a limit that was exceeded. The way this was passed before resulted in a broken error message like `"400 got more than Unknown bytes when reading 32768"` when it should have read `"400 got more than 32768 when reading request header"`.